pkg/liqonet/iptables/iptables.go

```diff
@@ -667,6 +923,7 @@ func (h IPTHandler) updateSpecificRulesPerChain(chain string, existingRules []st
 		for _, newRule := range newRules {
 			if strings.Contains(existingRule.String(), newRule.String()) {
 				outdated = false
+				break
```

Comment:
Because in that inner loop you are searching for a match of a specific existing rule (given from outer loop) with new rules: if you find that, rule is not outdated, so you can break